web/src/features/fba/components/ZoneSummaryPanel.tsx

```diff
@@ -1,30 +1,183 @@
 import React from 'react'
-import { TextField } from '@mui/material'
+import { Grid, Paper, Typography } from '@mui/material'
 import makeStyles from '@mui/styles/makeStyles'
-import { isNull } from 'lodash'
+import { isUndefined } from 'lodash'
+import { FireZone, FireZoneThresholdFuelTypeArea } from 'api/fbaAPI'
+import { PieChart, Pie, ResponsiveContainer, Cell } from 'recharts'
 
 const useStyles = makeStyles({
   wrapper: {
-    minWidth: 300
+    minWidth: 400
+  },
+  zoneName: {
+    fontSize: '2rem',
+    textAlign: 'center',
+    variant: 'h2'
+  },
+  centreName: {
+    fontSize: '1rem',
+    textAlign: 'right',
+    variant: 'h6'
+  },
+  fuelTypesPaper: {
+    padding: '20px 10px'
+  },
+  fuelTypesHeader: {
+    fontSize: '1.3rem',
+    textAlign: 'center',
+    variant: 'h3'
+  },
+  pieChartHeader: {
+    fontSize: '1rem',
+    variant: 'h4'
   }
 })
 
 interface Props {
   className?: string
-  selectedZoneID: number | null
+  selectedFireZone: FireZone | undefined
+  fuelTypeInfo: Record<number, FireZoneThresholdFuelTypeArea[]>
 }
 
+interface FuelTypeDataForPieChart {
+  area: number
+  fuel_type_code: string
+}
+
+const RADIAN = Math.PI / 180
+const COLOURS = [
+  '#2191FB',
+  '#FCB1A6',
+  '#B33951',
+  '#CCF5AC',
+  '#8CDEDC',
+  '#9DACFF',
+  '#4F7CAC',
+  '#FFA62B',
+  '#C09BD8',
+  '#EBC3DB',
+  '#D19C1D',
+  '#FFC0BE',
+  '#ED7D3A'
+]
+
 const ZoneSummaryPanel = (props: Props) => {
   const classes = useStyles()
 
-  if (isNull(props.selectedZoneID)) {
+  const renderCustomizedLabel = ({
+    cx,
```

Comment:
We should probably tell Typescript to ignore all the `any` types that `recharts` forces upon us here